src/Notifier.js

```diff
@@ -16,8 +16,10 @@ limitations under the License.
 
```

Comment:
Is there a reason for changing this into a non-replaceable component? Surely lots of apps will want to present their notifications differently?